webpack.config.js

```diff
@@ -31,94 +30,29 @@ const pathResolver = new PathResolver();
 const globber = new Globber({pathResolver});
 const pluginFactory = new PluginFactory({globber});
 const cssBundleFactory = new CssBundleFactory({env, pathResolver, globber, pluginFactory});
+const jsBundleFactory = new JsBundleFactory({env, pathResolver, globber, pluginFactory});
 
-const OUT_DIR_ABS = path.resolve('./build');
+const OUT_DIR_ABS = pathResolver.getAbsolutePath('./build');
```

Comment:
Are we still planning on sharing the CSS/JSBundleFactory methods outside of this repo? If not then I don't think `OUT_DIR_ABS` and `DEMO_ASSET_DIR_REL` needs to be passed in.